src/ert/config/everest_constraints_config.py

```diff
@@ -27,6 +27,10 @@ def metadata(self) -> list[ResponseMetadata]:
     type: Literal["everest_constraints"] = "everest_constraints"
     name: str = "everest_constraints"
     has_finalized_keys: bool = True
+    targets: list[float | None]
+    scales: list[float | None]
+    upper_bounds: list[float]
+    lower_bounds: list[float]
```

Comment:
Is this a candidate for a data structure? I assume the length of all these lists have to add up and that the indices matter.